src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
Discussed with Liam, we will not support TA based capacities just yet because it causes imbalance in invested time across TA's. Element of this decision is that it takes a bit of time to implement a proper UI for this that also ensures that there is enough capacity to distribute all groups, and that all team members currently have more important issues to work on. We definitely do want to support this feature at some point though, we're just not sure if all requirements are clear right now and how to implement it right.

Instead we will allow TA's to handle the situation themselves and pass on assigned groups to someone else. Therefore we added the ability to reassign a group.